lib/trocla.rb

```diff
@@ -31,7 +32,7 @@ def password(key,format,options={})
   end
 
   def get_password(key,format)
-    cache.fetch(key,{})[format]
+    decrypt cache.fetch(key,{})[format]
```

Comment:
could we have braces around the fetch? this would make it more readable, what belongs to the method call and not.